### DISCUSSION TITLE
Using DB like Cassandra in AppImage's way

### DISCUSSION THREAD

**Discussion by filosof86** [AppImage/AppImageKit#1332] [Q&A] [answered]
Hi all,

I wonder if it's possible (and advisable) to pack DBs like Cassandra in an AppImage and use it. 
I.e. is it ok to use the AppImage for serving the server-side applications?
Can it be easily deployed, accessed from the application outside the AppImage environment, etc?

Thank you in advance for your answers!

**Comment by probonopd** [comment, marked as answer]
Sure, that would be possible. There is no sandboxing in AppImage, so a database running from an AppImage can be accessed from applications outside of the AppImage just like normal.

AppImage is just a self-mounting disk image that executes whatever the author has put inside.
